scenarios/azure/aks/run.go

```diff
@@ -13,11 +25,105 @@ func Run(ctx *pulumi.Context) error {
 		return err
 	}
 
-	_, kubeConfig, err := aks.NewCluster(env, "aks", nil)
+	clusterComp, err := components.NewComponent(*env.CommonEnvironment, env.Namer.ResourceName("aks"), func(comp *kubeComp.Cluster) error {
+		cluster, kubeConfig, err := aks.NewCluster(env, "aks", nil)
+		if err != nil {
+			return err
+		}
+
+		ctx.Export("kubeconfig", kubeConfig)
+		// Filling Kubernetes component from EKS cluster
+		comp.ClusterName = cluster.Name
+		comp.KubeConfig = kubeConfig
+
+		// Building Kubernetes provider
+		aksKubeProvider, err := kubernetes.NewProvider(env.Ctx, env.Namer.ResourceName("k8s-provider"), &kubernetes.ProviderArgs{
```

Comment:
The provider needs to be store inside `comp`.

Reply:
I didn't see it for other scenarios but I added it now